v4/export/prepare.go

```diff
@@ -98,11 +99,37 @@ func filterDirtySchemaTables(conf *Config) {
 	switch conf.ServerInfo.ServerType {
 	case ServerTypeTiDB:
 		for dbName := range conf.Tables {
-			switch strings.ToUpper(dbName) {
-			case "INSPECTION_SCHEMA", "METRICS_SCHEMA", "PERFORMANCE_SCHEMA", "INFORMATION_SCHEMA":
+			switch strings.ToLower(dbName) {
+			case util.InformationSchemaName.L, util.PerformanceSchemaName.L, util.MetricSchemaName.L, util.InspectionSchemaName.L:
```

Comment:
Perhaps just use https://pkg.go.dev/github.com/pingcap/tidb-tools/pkg/filter?tab=doc#IsSystemSchema as we're now depending on TiDB anyway


```suggestion
			if filter.IsSystemSchema(strings.ToLower(dbName)) {
```